auth-web/src/services/business.services.ts

```diff
@@ -206,6 +206,28 @@ export default class BusinessService {
     })
   }
 
+  /**
+   * Get download url from Document Record Service
+   * @param documentServiceId the unique id on Document Record Service
+   * @param documentClass the document class defined for the document service. e.g. 'CORP'
+   * @returns a promise to return the string of file download.
+   */
+  static async getDownloadUrl (documentKey: string, documentClass: string): Promise<string> {
+    // safety checks
+    if (!documentKey || !documentClass) throw new Error('Invalid parameters')
+
+    const url = `${ConfigHelper.getLegalAPIV2Url()}/documents/drs/${documentClass}/${documentKey}`
+
+    return axios.get(url).then(response => {
```

Comment:
.then(), when we're using try catch async await below - seems weird but that's ok